types/react-spotify-auth/index.d.ts

```diff
@@ -0,0 +1,57 @@
+// Type definitions for react-spotify-auth 1.4
+// Project: https://github.com/kevin51jiang/react-spotify-auth
+// Definitions by: jsarlo <https://github.com/DefinitelyTyped>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 4.8
+
+import Window = require('trusted-types');
```

Comment:
where this bit came from? The native package seems to not have this very specific dependency